.github/workflows/build.yml

```diff
@@ -0,0 +1,172 @@
+name: Build
+
+on:
+  push:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-web:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions/setup-node@v1
+        name: Setup Node
+        with:
+          node-version: "*"
+      - name: Install Yarn
+        run: corepack enable
+      - name: Install npm dependencies
+        run: yarn install
+      - name: Build
+        run: yarn run build
+      - uses: actions/upload-artifact@v3
+        name: Upload Artifacts
+        with:
+          name: tauri-web
+          path: dist/*
+  build-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions-rs/toolchain@v1
+        name: Setup Rust
+        with:
```

Comment:
Wouldn't the minimal profile be sufficient? Is the toolchain cached?

Reply:
I don't know if it is cached (should be done by microsoft). There is a version of rust preinstalled in the runner image, but for consistency and newest updates it is practice to install it manually.

Reply:
Definitely, I meant the install files. `actions-rs` isn't directly by GitHub/Microsoft AFAIK. `setup-node` takes care of this internally, so probably `toolchain` does so too